main.go

```diff
@@ -34,6 +36,10 @@ func main() {
 		log.Println(err)
 	}
 
+	if _, err := nc.Subscribe("service.import.done", importDoneHandler); err != nil {
```

Comment:
Shouldn't this stuff be on the [definition-mapper](https://github.com/ErnestIO/definition-mapper) service instead? Otherwise it will be catching all providers events.

An easy approach is make this listen to `service.import.aws.done` and `service.import.aws.error`, while you add the routing listeners for `service.import.done` and `service.import.error` on the definition-mapper service. That will show continuity with current approach, while keeps your code on the same place :)